### DISCUSSION TITLE
Allow for comments on NFTs via replies

### DISCUSSION THREAD

**Discussion by erikreppel** [farcasterxyz/protocol#77] [FIP Stage 1: Ideas]
Was looking through the protobuf definitions and it occurred to me it would be really interesting to be able to reply to things other than a `CastId`. Ex: replying to an NFT could be the equivalent of commenting on an IG photo.

Change needed in the protocol:
```
oneof parent {
    CastId parent_cast_id = 3; // Parent cast of the cast
    String nft_id = 6; // x-chain identifier for an nft (ex: nft://<network>/<chain_id>/<address>/<token_id>)
  };
```

In the future might be interesting to open up commenting on transactions and other things too.
https://github.com/farcasterxyz/hub-monorepo/blob/main/protobufs/schemas/message.proto#L103

One standard for identifying NFTs cross chain https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md

**Comment by pfletcherhill** [comment]
The plan is to implement this as part of the next protocol release. See the discussion here: https://github.com/farcasterxyz/protocol/discussions/71

Hub implementation: https://github.com/farcasterxyz/hub-monorepo/issues/845

**Comment by varunsrin** [comment]
Closing as duplicate
